astropy/coordinates/tests/test_frames_with_velocity.py

```diff
@@ -272,14 +272,12 @@ def test_shorthand_attributes():
         pm_dec=np.random.normal(0, 100, n) * u.mas / u.yr,
         radial_velocity=np.random.normal(0, 100, n) * u.km / u.s,
     )
-    v = icrs1.velocity
```

Comment:
this test is about checking that attributes are accessible, so this line shouldn't be removed. One way to keep linters happy while still exercising this functionality is to prefix the variable name as
```python
    _v = icrs1.velocity
```

Reply:
Doh! Thanks for catching this, and for teaching me to placate the linter with an underscore, I didn't know that was possible.

Reply:
I added `hypothesis` tests mimicking what was there before, let me know what you think. I don't have enough domain knowledge yet to feel confident in rewriting the entire test.

Reply:
I'm not sure if this test needs to be implemented as a property-based test, but if it is to be one then the updated code is much better than the original.